db/fixtures/ae_datastore/ManageIQ/System/CommonMethods/QuotaMethods.class/__methods__/validate_quota.rb

```diff
@@ -80,13 +80,14 @@ def check_quota_results
   unless @max_exceeded.empty?
     max_message = message_text(nil, "Request exceeds maximum allowed for the following: ", @max_exceeded)
     message = set_exceeded_results(message, max_message, :quota_max_exceeded, "error")
+    @miq_request.set_message(message[0..250])
```

Comment:
Trimming the message length should be pushed down into the service model where the `set_message` method is defined so that it is performed for any caller.

The `message` variable that is defined outside of this block on line 79 is not needed since it is not used outside the block anymore.  Also, the `set_exceeded_results` is taking the `message` variable, which is always empty, and appending the second parameter string.  I do not see the purpose of that logic now or with the previous code.  Did this have any purpose at one time?

This code could be changed, if the first unused parameter in `set_exceeded_results` is dropped:
```ruby
@miq_request.set_message(set_exceeded_results(max_message, :quota_max_exceeded, "error"))
```